### PR TITLE
Use log crate

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

**Comment by dns2utf8** [comment]
:rainbow:

File: src/main.rs [hunk 1]
```diff
@@ -6,6 +6,9 @@ extern crate telegram_bot;
 extern crate hyper;
 extern crate rustc_serialize;
 extern crate spaceapi;
+extern crate env_logger;
+#[macro_use]
```
Comment (dns2utf8) [inline src/main.rs]: Does this import Macros from `log`?

Comment (rnestler) [inline src/main.rs]: Yes

**Comment by rnestler** [comment]
Do you think we can merge this?

**Comment by dns2utf8** [comment]
@rnestler Checking…

File: src/main.rs (hunk 1 above)
Comment (rnestler) [inline src/main.rs]: We could rewrite it on the same line if you think it's nicer:

``` Rust
#[macro_use] extern crate log;
```

Comment (dns2utf8) [inline src/main.rs]: ah, learned somthing new :smile: 

this woud be nicer :grin:

Comment (rnestler) [inline src/main.rs]: Done :smile:

**Comment by dns2utf8** [comment]
:rainbow: you may merge 

Thanks

**Pull request merged**